index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="personal, webpage, Christophers website" content="Christopher's personal webpage showcasing his Professional portfolio, biography, and contact information.">
+    <link rel="stylesheet" href="styles.css" />
+    <script src="https://kit.fontawesome.com/c740582a04.js" crossorigin="anonymous"></script>
+    <title>Christopher's Webpage</title>
+</head>
+<body>
+    <header>
+    <main>
+        <nav>
+            <ul>
+                <li id="home-nav"><a href="#" id="back-to-top">Home</a></li>
+                <li><a href="#bio">Bio</a></li>
```

Comment:
hmm, even though this list starts with Bio first, the browser shows Contact farthest to the left, when I was expecting Bio to be. Is this intentional?